src/main/java/com/zotov/edu/passportofficerestservice/repository/PersonsRepositoryCollections.java

```diff
@@ -0,0 +1,54 @@
+package com.zotov.edu.passportofficerestservice.repository;
+
+import com.zotov.edu.passportofficerestservice.repository.entity.Person;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Repository;
+
+import java.util.*;
+
+@Repository
+public class PersonsRepositoryCollections implements PersonsRepository {
+
+    NavigableMap<String, Person> persons = new TreeMap<>();
```

Comment:
It seems there's no reason to have `NavigableMap` here

Reply:
It has been changed to Map.